frontend/src/components/user/fields/SearchSelectorField.tsx

```diff
@@ -0,0 +1,246 @@
+import { QuestionOutlineIcon, ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Container,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Tag,
+  TagCloseButton,
+  Text,
+  Tooltip,
+} from "@chakra-ui/react";
+import React, { useRef, useState } from "react";
+import TagsPopover from "../../common/TagsPopover";
+
+export type Item = {
```

Comment:
I think we temporarily used the term "setting"(s) which applies to skills, languages, and branches (eg: SettingManager). Maybe we can keep it consistent.

```suggestion
export type Setting = {
```

Reply:
Oh I see, tbh I would want to keep it as "Item" since this is what we used in `SelectorField.tsx`, so I'd rather be consistent with that.

Reply:
Ah gotchu. Yea let's stick with item then.